File: 2026-01/src/day_31/boj_10178/Main.java
package day_31.boj_10178;

import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();
        StringTokenizer st;

        int t = Integer.parseInt(br.readLine());
        while (t-- > 0) {
            st = new StringTokenizer(br.readLine());
            int c = Integer.parseInt(st.nextToken());
            int v = Integer.parseInt(st.nextToken());

            sb.append("You get ").append(c / v).append(" piece(s) and your dad gets ").append(c % v).append(" piece(s).\n");
        }

        System.out.println(sb);
    }
}

File: 2026-01/src/day_31/boj_10178/Main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    ios::sync_with_stdio(0);
    cin.tie(0);

    int t;
    cin >> t;

    while (t--) {
        int c, v;
        cin >> c >> v;

        cout << "You get " << (c / v) << " piece(s) and your dad gets " << (c % v) << " piece(s).\n";
    }
}
